<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QDataTypeEditor</class>
 <widget class="QDialog" name="QDataTypeEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Редактор типа данных</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/G.png</normaloff>:/images/G.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>241</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <widget class="QLabel" name="typeNameLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Имя типа:</string>
    </property>
    <property name="buddy">
     <cstring>typeNameEdt</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="typeNameEdt">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>291</width>
      <height>20</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="readOnly">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="typedefLbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>167</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Определение типа (typedef)</string>
    </property>
    <property name="buddy">
     <cstring>typedefTxtEdt</cstring>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="typedefTxtEdt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>361</width>
      <height>171</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="tabChangesFocus">
     <bool>true</bool>
    </property>
    <property name="backgroundVisible">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>typeNameEdt</tabstop>
  <tabstop>typedefTxtEdt</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>QDataTypeEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>QDataTypeEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>typeNameEdt</sender>
   <signal>textChanged(QString)</signal>
   <receiver>QDataTypeEditor</receiver>
   <slot>enableOkButton()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>184</x>
     <y>31</y>
    </hint>
    <hint type="destinationlabel">
     <x>19</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>enableOkButton()</slot>
 </slots>
</ui>
